### ISSUE TITLE
Unexpected format of `Include` in `--show-cops` when using configuration inheritance

### ISSUE THREAD

**Issue by leoarnold** [rubocop/rubocop#11398]
Given the two configuration files

```yaml
# cat .rubocop.yml 
AllCops:
  NewCops: enable
  DisabledByDefault: true

Style/FrozenStringLiteralComment:
  Enabled: true
```

and

```yaml
# lib/.rubocop.yml 
inherit_from: '../.rubocop.yml'

Style/FrozenStringLiteralComment:
  Include:
    - '*.rb'
```

## Expected behavior

In the `lib` directory:

```
(lib) $ rubocop --show-cops Style/FrozenStringLiteralComment
# Supports --autocorrect
Style/FrozenStringLiteralComment:
  Description: Add the frozen_string_literal comment to the top of files to help transition
    to frozen string literals by default.
  Enabled: true
  VersionAdded: '0.36'
  VersionChanged: '0.79'
  EnforcedStyle: always
  SupportedStyles:
  - always
  - always_true
  - never
  SafeAutoCorrect: false
  Include:
  - "lib/*.rb" <---
```

## Actual behavior

In the `lib` directory:

```
(lib) $ rubocop --show-cops Style/FrozenStringLiteralComment
# Supports --autocorrect
Style/FrozenStringLiteralComment:
  Description: Add the frozen_string_literal comment to the top of files to help transition
    to frozen string literals by default.
  Enabled: true
  VersionAdded: '0.36'
  VersionChanged: '0.79'
  EnforcedStyle: always
  SupportedStyles:
  - always
  - always_true
  - never
  SafeAutoCorrect: false
  Include:
  - "../*.rb" <--- 
```

## RuboCop version

```
$ rubocop -V
1.42.0 (using Parser 3.2.0.0, rubocop-ast 1.24.1, running on ruby 3.1.3) [x86_64-linux]
```

**Comment by jonas054** [comment]
I found the problem. A fix is coming.
